app/templates/select-item-layout.hbs

```diff
@@ -1,5 +1,11 @@
 {{#if view.content.isGroupOption}}
   {{view.content.name}}
 {{else}}
-  {{yield}}
```

Comment:
True, that's probably what caused that one regression we fixed. I don't remember why this was causing issues, I think it was because we were relying on both `templateName` and `layoutName` to work together and that doesn't work anymore. In other words, you _must_ use block syntax in a component for `{{yield}}` to work.